libraries/crypto/src/hkdf.rs

```diff
@@ -77,4 +94,39 @@ mod test {
             );
         }
     }
+
+    #[test]
+    fn test_hkdf_256_sha256_vectors() {
+        // Test vectors generated as above, but with salt:
+        let test_okms = [
+            hex::decode("f9be72116cb97f41828210289caafeabde1f3dfb9723bf43538ab18f3666783a")
+                .unwrap(),
+            hex::decode("a2480a09c7349d76e459f98a8259da40544bfbd2930d357a0f3250ade0acf941")
+                .unwrap(),
+            hex::decode("3904f7bf3615df9512fc6b1af651ed69b43f7fad424f9c718aaab63f377a36b9")
+                .unwrap(),
+            hex::decode("a0027dcffb27d356317199c6e65f153a9286ba114aee2d3cf45bdba83cb7c065")
+                .unwrap(),
+            hex::decode("786d1f89f54668bac443cc6a8887c95d6fbde07702cb4c16d76c452e87c50f79")
+                .unwrap(),
+            hex::decode("8e9a5bdf362c5aec2c31a742dfebd0b7b56e16ab8408d9d0609a4fad06446875")
+                .unwrap(),
+            hex::decode("4a35d3d7c80ff4fab65f7e30d6b305fc7bb39ffe905aabedd6593354f86177b6")
+                .unwrap(),
+            hex::decode("b1121deabd8b4308f3805cda8af991ee976bd8e413bcb6a8dd3fc26ebe2312d2")
+                .unwrap(),
```

Comment:
The `hex::decode()` could be done in the loop to minimize boilerplate.

Reply:
Done. Also I forgot to push my last change. The `rust_crypto` feature is only tested locally, not in workflows, as it is more of an example. In case you are wondering why CI didn't notice on my first commit.